open_source_maintenance_policy.md

```diff
@@ -0,0 +1,138 @@
+This is adapted from the [Chef Software](https://github.com/chef/chef-rfc/blob/master/rfc030-maintenance-policy.md#how-the-project-is-maintained) Open Source Maintenance Policy.
+
+# Maintenance Policy
+
+The Maintenance Policy defines how we make decisions about what happens with Operation Code, and associated software projects. It provides the process by which:
+
+* Roadmaps are decided
+* Patches are merged
+* Disputes are resolved
+
+It is intended to be short, flexible, and clear.
+
+This file is related to the MAINTAINERS and ROADMAP files in Operation Code projects.
+
+# How the project is maintained
+
+This file is the canonical source for how the Operation Code projects are maintained.
+
+# Roles
+
+## Project Lead
+
+* Resolves disputes
+* Provides vision and roadmap
+* Has universal veto power
+* There can be only one
+
+## Lieutenant
+
+* Each component in the project may have at most one Lieutenant
+* Provides guidance on future direction for their component
+* Resolves disputes within their component
+* Has localized veto power
+* Plus all the responsibilities of a Maintainer
+
+## Maintainer
+
+* Each component may have multiple Maintainers
+* Handles contributions on GitHub - first response to a PR within 48 hours
+* Is available in Slack
+* Committed to 100% tests passing for your component
+* Has full commit/merge access to the relevant repositories
+
+# Roadmap
+
+All project roles should work together to determine the best opportunities for the improvement of the project. This should produce direction for the project, which should be highlighted in the ROADMAP.
+
+As a project role does not necessarily control resources other than their own labor, they should encourage contributors to participate in furthering development of the project toward the goals of the ROADMAP.
+
+Releases should not block on features on the roadmap, rather they should happen as features are completed. The ROADMAP should be organized by general time periods, not by versions
+## Example Roadmap entries
+
+```
+# 2015 Q2
+* Component - Roadmap Item
+  Description
+
+# 2015 Q4
+* Core - Support for Drone Automation
+  Significant time could be saved in enterprise datacenters by using drones to stack and rack servers
+
+# 2016 Q1
+* Core - Support Internet of Things
+  As a Chef user, I look forward to the singularity
+```
+
+# Contributing Patches
+
+## How a patch gets merged
+
+* Open Pull Request (anyone)
+* Code reviewed by a Maintainer, Lieutenant, or Project Lead. Approval is indicated by :+1: on the pull request.
```

Comment:
At my company, we lean on GitHub's new'ish **Review Changes** feature here, where the PR would need to be **approved** instead of "+1".  This also dove tails well with the OC's current merge requirements, where a PR has to be **approved** through this feature, to be mergeable.  

Perhaps we can leverage that feature here, as well?

Reply:
Absolutely!  That's a great idea, updating :)